CITIES.md

```diff
@@ -13,7 +13,7 @@ The municipality id (IBGE code) can be found on [Wikipedia](https://pt.wikipedia
 | 3 | Brasília | :soon: | | | [PR](https://github.com/okfn-brasil/diario-oficial/pull/57) |
 | 4 | Salvador | :soon: | | | [PR](https://github.com/okfn-brasil/diario-oficial/pull/47) |
 | 5 | Fortaleza | :soon: | | | [PR](https://github.com/okfn-brasil/diario-oficial/pull/52) |
-| 6 | Belo Horizonte | :soon: | | | [PR](https://github.com/okfn-brasil/diario-oficial/pull/33) |
+| 6 | Belo Horizonte | :white_check_mark: | | | [PR](https://github.com/okfn-brasil/diario-oficial/pull/33) |
```

Comment:
I think the URL must be updated as well. :)

Reply:
I still think that most of the work was done by @juniorcarvalho – and at the end of the discussion there, reader is linked to this one. So I prefer to link to @juniorcarvalho's PR. Does that make sense for anyone else or is it just me?

Reply:
I thought about it at first glance but it would make sense to have the reference to the code the was merged as well (and it's clear in the description of this one that @juniorcarvalho did it). :) But nothing serious, anyway.